### ISSUE TITLE
Low performance of Pedstrian on the test set and val set of KITTI.

### ISSUE THREAD

**Issue by Blinkblade** [open-mmlab/mmdetection3d#2904]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection3d/issues) and [Discussions](https://github.com/open-mmlab/mmdetection3d/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection3d.readthedocs.io/en/latest/notes/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (dev-1.x)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x) or [latest version (dev-1.0)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.0).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

main branch https://github.com/open-mmlab/mmdetection3d

### Environment

sys.platform: linux
Python: 3.8.10 (default, Jun  4 2021, 15:09:15) [GCC 7.5.0]
CUDA available: True
numpy_random_seed: 2147483648
GPU 0: Tesla V100S-PCIE-32GB
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 11.3, V11.3.109
GCC: gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
PyTorch: 1.11.0+cu113
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.5.2 (Git Hash a9302535553c73243c632ad3c4c80beec3d19a1e)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.3
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.2
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.3, CUDNN_VERSION=8.2.0, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.11.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.12.0+cu113
OpenCV: 4.8.0
MMEngine: 0.8.4
MMDetection: 3.1.0
MMDetection3D: 1.2.0+0f9dfa9
spconv2.0: True

### Reproduces the problem - code sample

official code

### Reproduces the problem - command or script

python tools/train.py configs/parta2/parta2_hv_secfpn_8xb2-cyclic-80e_kitti-3d-3class.py 

### Reproduces the problem - error message

Hello,

I've been working on 3d detection using the KITTI dataset with several models (PointPillars, SECOND, PartA2) provided by MMDetection3D. However, I've encountered a challenge where the AP of Pedestrians for these models seems to plateau around 60%(for the easy cases on KITTI val set), which is significantly lower than the performance reported in the original papers (such as PartA2 ).  And when I submit the results on KITTI test server, the Pedestrians AP is significantly lower. For example, the result of PartA2 I submit is:

Benchmark	Easy	Moderate	Hard
Car (Detection)	95.83 %	92.12 %	89.36 %
Car (Orientation)	95.43 %	91.39 %	88.52 %
Car (3D Detection)	87.42 %	76.05 %	71.31 %
Car (Bird's Eye View)	92.55 %	87.63 %	83.03 %
Pedestrian (Detection)	61.10 %	50.01 %	47.44 %
Pedestrian (Orientation)	55.70 %	45.27 %	42.77 %
Pedestrian (3D Detection)	44.38 %	36.14 %	33.27 %
Pedestrian (Bird's Eye View)	48.90 %	41.10 %	38.83 %
Cyclist (Detection)	88.12 %	74.99 %	69.17 %
Cyclist (Orientation)	87.83 %	74.37 %	68.48 %
Cyclist (3D Detection)	78.05 %	60.23 %	54.94 %
Cyclist (Bird's Eye View)	80.61 %	63.20 %	57.70 %

but the original PartA2 is :
Benchmark	Easy	Moderate	Hard
Car (Detection)	95.03 %	91.86 %	89.06 %
Car (Orientation)	95.00 %	91.73 %	88.86 %
Car (3D Detection)	87.81 %	78.49 %	73.51 %
Car (Bird's Eye View)	91.70 %	87.79 %	84.61 %
Pedestrian (Detection)	68.78 %	57.96 %	54.01 %
Pedestrian (Orientation)	63.51 %	52.20 %	48.27 %
Pedestrian (3D Detection)	53.10 %	43.35 %	40.06 %
Pedestrian (Bird's Eye View)	59.04 %	49.81 %	45.92 %
Cyclist (Detection)	88.90 %	78.29 %	71.19 %
Cyclist (Orientation)	88.70 %	77.52 %	70.41 %
Cyclist (3D Detection)	79.17 %	63.52 %	56.93 %
Cyclist (Bird's Eye View)	83.43 %	68.73 %	61.85 %

The same problem also occurred when experimenting with SECOND and PointPillars. I've tried various strategies to address this, including modifying the version of mmdetection3d, tuning hyperparameters, and regenerating data.

Is lower detection AP for Pedestrians a common phenomenon on KITTI in MMDetection3D?

Could you please advise if there are any known best practices or suggestions for improving this within the MMDetection3D framework? Additionally, I'm curious if others have experienced similar issues and how they might have overcome them.

Thanks for any help!

**Comment by Gaviaa** [comment]
I also encountered this problem. Have you solved it?

**Comment by Blinkblade** [comment]
> I also encountered this problem. Have you solved it?
no idea about it
